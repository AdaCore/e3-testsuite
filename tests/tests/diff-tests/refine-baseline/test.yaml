driver: diff-script-driver
process_args: [-phello, -pworld, -pactual_to_refine]
refine_baseline: True
