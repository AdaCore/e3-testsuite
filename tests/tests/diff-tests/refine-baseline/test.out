hello
world
baseline_to_refine
